appveyor.yml

```diff
@@ -1,3 +1,6 @@
+skip_commits:
+  message: /^Merge pull request /
```

Comment:
Could add `author: noreply@github.com` or similar to restrict this to PRs merged through the GitHub interface. Not sure it is worth it, but figured it deserved a mention.